apix-impl/src/main/java/eu/xenit/apix/alfresco/dictionary/AspectService.java

```diff
@@ -56,6 +58,23 @@ public AspectDefinition GetAspectDefinition(QName qname) {
         return GetAspectDefinition(c.alfresco((qname)));
     }
 
+
+    @Override
+    public Aspects getAspects() {
+        Collection<org.alfresco.service.namespace.QName> aspects = dictionaryService
+                .getAllAspects();
+        List<AspectDefinition> ret = new ArrayList<>(aspects.size());
+        for (org.alfresco.service.namespace.QName aspect : aspects) {
+
+            AspectDefinition aspectDefinition = GetAspectDefinition(aspect);
+            assert aspectDefinition != null;
+            ret.add(aspectDefinition);
+        }
+        return new Aspects(ret);
+
+
+        }
```

Comment:
@rawiabenhmida Can you remove the new lines at the start of the for loop and underneath `return new Aspects(ret);`?
Can you also move the last "}" at the end of the method 1 tab to the left?